src/main/java/mate/controller/cars/AddDriverController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller.cars;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class AddDriverController extends HttpServlet {
+    private static final Injector INJECTOR =
+            Injector.getInstance("mate");
+    private static final DriverService DRIVER_SERVICE =
+            (DriverService) INJECTOR.getInstance(DriverService.class);
+
+    private static final CarService CAR_SERVICE =
+            (CarService) INJECTOR.getInstance(CarService.class);
```

Comment:
```suggestion
    private final DriverService driverService =
            (DriverService) INJECTOR.getInstance(DriverService.class);

    private final CarService carService =
            (CarService) INJECTOR.getInstance(CarService.class);
```

services should not be static, fix all controllers